### DISCUSSION TITLE
shap.summary_plot() source code

### DISCUSSION THREAD

**Discussion by TrianaForment** [shap/shap#3699] [Q&A] [answered]
Hi everyone, 

I'm using the SHAP version 0.40.0 to explain RF feature importance. My piece of code is:

    explainer = shap.TreeExplainer(final_model, data = background, feature_perturbation = 'interventional')
    shaps = explainer.shap_values(background, check_additivity=False)
    
  Then I can plot the beeswarm plot as: shap.summary_plot(shaps[1], background, max_display=10)
  
 I don't find anywhere the source code of that function that produces the nice beeswarm plot. Can you help me?

**Comment by chadmhicks** [comment, marked as answer]
shap/plots/_beeswarm.py -> beeswarm()
